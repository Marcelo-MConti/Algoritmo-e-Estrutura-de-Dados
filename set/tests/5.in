1
70 150
100 98 97 96 92 91 89 88 87 85 84 83 82 79 76 75 74 73 71 70 69 65 64 63 59 58 57 55 53 49 47 45 44 41 40 39 37 34 32 29 28 27 25 21 18 16 12 5 4 2 0 -3 -4 -5 -6 -7 -9 -11 -12 -13 -15 -16 -17 -20 -23 -24 -26 -28 -29 -30
171 117 71 43 42 85 90 194 150 49 163 174 181 57 87 123 37 193 99 151 56 54 59 20 152 137 176 62 89 198 26 7 44 191 86 127 109 41 47 67 106 102 168 180 160 177 146 166 13 131 134 4 3 128 17 197 122 118 133 52 73 78 1 83 188 175 105 192 149 153 60 185 88 28 121 10 48 170 12 119 112 34 79 104 11 136 40 108 53 142 184 178 164 35 18 93 158 196 199 143 182 156 5 55 77 27 172 92 80 147 45 157 38 96 23 33 161 94 0 120 91 8 186 61 111 36 132 113 15 125 75 69 124 200 9 100 103 179 107 19 30 51 167 29 189 82 129 6 76 64
3
