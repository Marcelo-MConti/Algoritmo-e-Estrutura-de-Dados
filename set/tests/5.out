0, 4, 5, 12, 18, 27, 28, 29, 34, 37, 40, 41, 44, 45, 47, 49, 53, 55, 57, 59, 64, 69, 71, 73, 75, 76, 79, 82, 83, 85, 87, 88, 89, 91, 92, 96, 100, 
